MathComponent Tutorial - missing CMD_SEQ macro definition for testThrottle()

| | |
|:---|:---|
|**_F´ Version_**| head devel |
|**_Affected Component_**| MathComponent Tutorial |
---
## Problem Description

There is an issue in the section [5.5.3. Write and Run Tests](https://github.com/nasa/fprime/blob/devel/docs/Tutorials/MathComponent/Tutorial.md#553-write-and-run-tests) of the MathComponent tutorial.

Indeed, in the code snippet of the ``testThrottle()``, the symbol ``CMD_SEQ`` is not defined by default.

```bash
$ fprime-util build --ut
[  0%] Built target Fw_Cfg
[ 10%] Built target codegen
[ 20%] Built target Fw_Types
[ 20%] Built target Fw_Logger
[ 20%] Built target Fw_Obj
[ 30%] Built target Fw_Port
[ 30%] Built target Fw_Time
[ 30%] Built target Fw_Com
[ 40%] Built target Fw_Tlm
[ 50%] Built target Fw_Log
[ 60%] Built target Fw_Cmd
[ 60%] Built target Fw_Prm
[ 60%] Built target Ref_MathTypes
[ 60%] Built target Ref_MathPorts
[ 60%] Built target Utils_Hash
[ 60%] Built target Fw_Comp
[ 80%] Built target Os
[ 80%] Built target Fw_CompQueued
[ 90%] Built target Svc_Sched
[ 90%] Built target Ref_MathReceiver
[ 90%] Built target gtest
[100%] Built target gtest_main
[100%] Built target STest
Scanning dependencies of target Ref_MathReceiver_ut_exe
[100%] Building CXX object Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/test/ut/main.o
[100%] Building CXX object Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/test/ut/Tester.o
/mnt/c/Users/ThibFrgsGmz/fprime_lab/fprime/Ref/MathReceiver/test/ut/Tester.cpp: In member function ‘void Ref::Tester::testThrottle()’:
/mnt/c/Users/ThibFrgsGmz/fprime_lab/fprime/Ref/MathReceiver/test/ut/Tester.cpp:335:52: error: ‘CMD_SEQ’ was not declared in this scope
  335 |       this->sendCmd_CLEAR_EVENT_THROTTLE(INSTANCE, CMD_SEQ);
      |                                                    ^~~~~~~
make[3]: *** [Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/build.make:111: Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/test/ut/Tester.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:31252: Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:31259: Ref/MathReceiver/CMakeFiles/Ref_MathReceiver_ut_exe.dir/rule] Error 2
make: *** [Makefile:10939: Ref_MathReceiver_ut_exe] Error 2
[ERROR] CMake erred with return code 2
```

Unlike the ``INSTANCE`` macro, the ``CMD_SED`` symbol is not defined by default in the ``Tester.cpp`` file.

## How to Reproduce

1. Follow the MathComponent tutorial from ["Write and Run Unit Tests"](https://github.com/nasa/fprime/blob/devel/docs/Tutorials/MathComponent/Tutorial.md#55-write-and-run-unit-tests)
2. Reach the "Writing an Accelerator Test" section
3. Run ``fprime-util build --ut``.

## Expected Behavior

To solve this issue, the tutorial should inform the user to manually set the macro at the top of the ``TestER.cpp`` file:

```cpp
#define CMD_SEQ 0
```

Either the macro must be generated automatically in the same way as the ``INSTANCE`` macro.

--- a/docs/Tutorials/MathComponent/Tutorial.md
+++ b/docs/Tutorials/MathComponent/Tutorial.md
@@ -1829,7 +1829,7 @@ uses DIV instead of SUB.
 Add the following to the top of the `Tester.cpp` file:
 
 ```C++
-#define CMD_SEQ 0
+#define CMD_SEQ 42
 ```
 
 Add the following function to the "Tests" section of `Tester.cpp`:

--- a/docs/Tutorials/MathComponent/Tutorial.md
+++ b/docs/Tutorials/MathComponent/Tutorial.md
@@ -1826,6 +1826,12 @@ This test is the same as the SUB test, except that it
 uses DIV instead of SUB.
 
 **Write a throttle test:**
+Add the followint to the top of the `Tester.cpp` file:
+
+```C++
+#define CMD_SEQ 0
+```
+
 Add the following function to the "Tests" section of `Tester.cpp`:
 
 ```c++

--- a/docs/Tutorials/MathComponent/Tutorial.md
+++ b/docs/Tutorials/MathComponent/Tutorial.md
@@ -1826,7 +1826,7 @@ This test is the same as the SUB test, except that it
 uses DIV instead of SUB.
 
 **Write a throttle test:**
-Add the followint to the top of the `Tester.cpp` file:
+Add the following to the top of the `Tester.cpp` file:
 
 ```C++
 #define CMD_SEQ 0

